models/project_factory.rb

```diff
@@ -36,7 +36,7 @@ def all_projects_as_type
   end
 
   def all_active_projects_as_type
-    Project.active.map { |project| as_type(project) }
+    Project.where(active: 'true').in_date.map { |project| as_type(project) }
```

Comment:
The readme will need updating to reflect that we're now not just looking at end date.